pkg/config/kind.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package aspect
+package config
 
 // Kind of aspect
 type Kind int
```

Comment:
You use `Kind` as a `uint` everywhere, should we alias `uint` instead of `int` here?